src/Admin/AbstractAdmin.php

```diff
@@ -557,6 +557,8 @@ public function getDataSourceIterator()
 
     /**
      * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/admin-bundle 3.81, use configureFormOptions() to set $formOptions['constraints'] instead.
```

Comment:
3.82 according to the error in the method

Reply:
Sorry for that 😊 Fixed